mpt/impl/smtp/core/src/main/java/org/apache/james/mpt/smtp/ForwardSmtpTest.java

```diff
@@ -75,15 +76,17 @@ public void setUp() throws Exception {
         RestAssured.port = 80;
         RestAssured.baseURI = "http://" + containerIp.getHostAddress();
         RestAssured.config = newConfig().encoderConfig(encoderConfig().defaultContentCharset(Charsets.UTF_8));
+        RestAssured.requestSpecification = new RequestSpecBuilder()
+        		.setContentType(ContentType.JSON)
+        		.setAccept(ContentType.JSON)
+        		.build();
     }
 
     @Test
     public void forwardingAnEmailShouldWork() throws Exception {
         scriptTest("helo", Locale.US);
 
         given()
```

Comment:
Maybe here you can even remove the given() that do not add anything semantically...